### DISCUSSION TITLE
2009/09/11/%E6%83%B3%E7%9A%84%E8%B6%8A%E5%A4%9A%E8%B6%8A%E7%97%9B%E8%8B%A6

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#278] [Announcements]
# 2009/09/11/%E6%83%B3%E7%9A%84%E8%B6%8A%E5%A4%9A%E8%B6%8A%E7%97%9B%E8%8B%A6

最近这两天都有点失眠。昨晚我以为已经很累了，应当可以睡的比较长，结果今天早上3点钟的时候还是突然醒过来。天气变化的时候比较容易失眠而且这两天乱七八糟的事情想太多了。

https://blog.qizhen.xyz/2009/09/11/%E6%83%B3%E7%9A%84%E8%B6%8A%E5%A4%9A%E8%B6%8A%E7%97%9B%E8%8B%A6

**Comment by ruanqizhen** [comment]
之前博客下的留言：

彪
2010年02月11日 16:42 编辑
越想越痛苦，十分有九分是跟我一样的感觉吧！虽然我没有真正体验过工作的压力和痛苦还有快乐！像我们90后并且在校学习的学生，大部分学生都追捧电影明星、NBA男星、什么快乐男生、快乐女生等，常常将头挂在教室窗前，看着他们在操场上玩得那么开心，那么快乐！而自己还在心窝里闪烁着这个程序怎么设计？那道数学题怎么解？该玩的时候，常常因为自己的兴趣把头搞大了！这样也不错啊！最让人痛心的是：如果你要去想那些玩的人的心里是怎么想的？那才叫痛苦呢！因为你不知道他们是否为自己的未来奋力地拼搏，如果你站在他们的角度去思考那些所谓的理想、价值还有感情，我这头啊！比电脑死机还大，特别可悲的是在我们这一代的学生，越来越少有人进入科研的工作了！在进入大学以前，许多学生因为受到太多的“诱惑”决定加入娱乐圈，就不再培养自己刻苦专研的能力了！到了大学之后，猛然发现一切都太迟了，最后的结果是在中国土地上所作出的技术贡献越来越少，所以作为明志的我，常常一个人跳舞，难免不了多想啊！不过人生的意义不在于拥有什么，而在于留下什么！写几本书，在书上多写一些有关怎样培养学生刻苦专研精神的内容，我想到我们每一个engineer离开这个世界之前，回想一下一生中用自己务实的行动影响了后代，培养了他们对于科学积极地态度。我想那才叫过足了隐！

回复
阿七
2009年11月19日 10:42 编辑
这就好比设计一个程序，大的架构让人感到空虚绝望，但是没有大的架构又容易陷入无尽繁琐的局部变化。

回复
奕瑋
2009年09月29日 12:21 编辑
上次留言的时候忘记写上我另外一个观点了，那就是：过程往往比结果更重要。你今天看到你的著作得到认可，那是你十年来埋头苦干的结果，成就的喜悦是一瞬间，导致这个成就的过程却是漫长而卓绝的，这个过程也是更值得回味的。这只是打个比方说。所以我做事情，只要它是对的，为了别人好的，出发点是不错的，那就去做吧，一息尚存、锲而不舍，能做多少做多少，能做到什么成度就到什么成度。有点儿像愚公移山哈。

回复
yiwei
2009年09月28日 19:05 编辑
还是想得长远好，能做到把长远放在第一位不是很容易吧？

回复
茂
2009年09月24日 22:58 编辑
想得越多越痛苦，完全同意楼主的见地。真正的大师与先知多半是孤独的，甚至是痛苦的。佛学里有个“十四无难”的故事，也就是教导大家不要去探讨这些高深的问题，把眼睛蒙起来，做一个快乐的傻子：）

回复
奕瑋
2009年09月14日 10:59 编辑
有思想的人就是活的辛苦，尤其是百思不得其解的时候。不过这样也好，空活百年有啥意思呢？如果能在名利情之外看世界，从更宏观更微观的角度去看宇宙和天体；那么生命存在的形式和意义就截然不同了，再反过来看人类和人生可能就更不一样了。从这样一种超脱的态度来入世，我们的人生以及生命的意义会有全新的诠释，也会更有积极意义。现代的物理学家、天文学家也是人，也有他们难以逾越的境界；宗教界尽管有神传文化的传承，却鲜有实修者，而且现在也已经蜕变成人类社会的一种职业了，也是可悲。宇宙的起源与终结，大概只有成就这个宇宙的生命才能讲的清楚说的明白，而我们芸芸众生统统都是这个宇宙的造化。就像我们编的软件、造的机器，它们不会知道它们存在的意义，除非我们告诉它，可是我们如何告诉它呢？它们永远不会转化成为我们，甚至永远没有办法和我们直接沟通。生命是有层次和境界的，任何一个生命都有造就他生命的环境和因素，也因此从根本上限定了他的思维和特性，这就是定数。

回复
Benjamin
2009年09月14日 09:03 编辑
想得多，可是写的一点都不多。希望以后能多写些，把想的都写出来，呵呵~~

回复
li
2009年09月13日 20:31 编辑
享受过程,结果只是一刹那有了目标就去做撒~~~~想太多容易累

回复
竹贤
2009年09月12日 13:00 编辑
凡事如果能够考虑到5年、10年、更长一点20年的话，那就是长远考虑了。如果要探讨宇宙的起源与终结，那就是物理学家和宗教界考虑的事了。

回复
